### ISSUE TITLE
Wrap GenBank, FASTA, GFF3 SynBioHub download options

### ISSUE THREAD

**Issue by cjmyers** [SynBioDex/libSBOLj#605]
Add GenBank, FASTA, GFF3 SynBioHub download options to the SynBioHubFrontend.

**Comment by bbartley** [comment]
Hi,

Just noting, when I submit an SBOL2->GFF3 conversion job through the SBOL-Validator's http API, I receive the following response body:
```
libSBOLj version 2.3.0
Description: validates the contents of an SBOL 2.2.0 document, can compare two documents,
and can convert to/from SBOL 1, GenBank, FASTA, and SnapGene formats.

Usage:
	java --jar libSBOLj.jar [options] <inputFile> [-o <outputFile> -e <compareFile>]

Options:
	-l  <language> specfies language (SBOL1/SBOL2/GenBank/FASTA/SnapGene) for output (default=SBOL2)
	-s  <topLevelURI> select only this object and those it references
	-p  <URIprefix> used for converted objects
	-c  change URI prefix to specified <URIprefix>
	-v  <version> used for converted objects
	-t  uses types in URIs
	-n  allow non-compliant URIs
	-i  allow SBOL document to be incomplete
	-b  check best practices
	-f  fail on first error
	-d  display detailed error trace
	-mf main SBOL file if file diff. option is selected
	-cf second SBOL file if file diff. option is selected
	-no indicate no output file to be generated from validation
```
I do not receive this error with any other `language` options

**Comment by cjmyers** [comment]
Fixed by PR #609
